platform/nativescript/runtime/components/list-view.js

```diff
@@ -13,6 +20,9 @@ export default {
         templateSelector: {
             type: Function,
             default: () => 'default'
+        },
+        separatorColor: {
+            type: String
```

Comment:
Depends on how NativeScript handles empty values. If needed I think a `default` value is a better option thank making it `required`